modules/cluster-logging-elasticsearch-retention.adoc

```diff
@@ -0,0 +1,123 @@
+// Module included in the following assemblies:
+//
+// * logging/cluster-logging-elasticsearch.adoc
+
+[id="cluster-logging-elasticsearch-retention_{context}"]
+= Configuring log retention time
+
+You can specify how long the default Elasticsearch log store keeps indices 
+using a separate _retention policy_ for each of the three log sources: 
+infrastructure logs, application logs, and audit logs. The retention policy, 
+configured using the `maxAge` parameter in the Cluster Logging Custom Resource 
+(CR), determines when to delete the Elasticsearch indices. 
+
+If you do not create a retention policy, logs are retained for seven days by
+default. If you do not specify a retention policy for all three log sources,
```

Comment:
From where are you getting this information; it seems like a bug and I do not believe this to be correct if that's the behavior.  Retention policy is about keeping a certain number of TIMEUNIT of logs ... not about  logs being stored or not;  that is what logforwarding is about